src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -82,6 +82,8 @@ class DecodedResults {
     }
 };
 
+class LLMPipelineImplBase;
```

Comment:
I suppose we can move it to `LLMPipeline`'s private section

Reply:
It's an interface now, in case it's part of `LLMPipeline` it should be public.